### ISSUE TITLE
[BUG] ValueError: 'category' column dtypes are currently not supported by the gpu accelerated parquet writer

### ISSUE THREAD

**Issue by mlahir1** [rapidsai/cudf#10115]
The following exception is thrown when a column of type 'int' is written using to_parquet, and read with read_parquet and re-written with to-parquet again.

So, the first to_parquet is changing the `int` col type to catg, but when reading again, and the same file is re-written it fails.

Exception: ValueError: 'category' column dtypes are currently not supported by the gpu accelerated parquet writer

Reproducer Code:
```
import cudf
df = cudf.DataFrame()

df['a'] = [0 , 0 , 0, 1]
df['b'] = [0 , 0 , 0, 1]

df['a']  = df.a.astype('int')
df.to_parquet('test', partition_cols=['a'], partition_file_name=f'0.parquet')
df = cudf.read_parquet('test/a=0/0.parquet')
df.to_parquet('test.pq')
```

**Comment by Pedrexus** [comment]
any update on this part? Is it possible to be implemented?

**Comment by vyasr** [comment]
I don't believe that there has been any change here unfortunately. @vuule could you please confirm?
